proto/desmos/profiles/v1beta1/models.proto

```diff
@@ -125,24 +127,46 @@ message Proof {
 
 // ChainConfig contains the data of the chain with which the link is made.
 message ChainConfig {
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_stringer) = true;
+
   string name = 1
       [ (gogoproto.moretags) = "yaml:\"name\"" ];
-  string bech32_addr_prefix = 2
-      [ (gogoproto.moretags) = "yaml:\"bech32_addr_prefix\"" ];
 }
 
 // ChainLink contains the data representing either an inter- or cross- chain link
 message ChainLink {
-  string address = 1 [ (gogoproto.moretags) = "yaml:\"address\"" ];
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_stringer) = true;
+  
+  google.protobuf.Any address = 1 [ (gogoproto.moretags) = "yaml:\"address\"" ];
```

Comment:
Can we add the `cosmos_proto.accepts_interface` Protobuf option here the same way that [Cosmos does](https://github.com/cosmos/cosmos-sdk/blob/master/proto/cosmos/auth/v1beta1/query.proto#L40)? I think it would help us remember which interface should be used in the future and for other developers

```protobuf
  google.protobuf.Any address = 1 [ (gogoproto.moretags) = "yaml:\"address\"", (cosmos_proto.accepts_interface) = "AddressData" ];
```